project/backend/user-service/docker-compose.yml

```
@@ -0,0 +1,28 @@
+      # Use standard Spring Boot environment variable names
+      - SPRING_DATASOURCE_URL=jdbc:mysql://host.docker.internal:3306/metadocs
+      - SPRING_DATASOURCE_USERNAME=root # <<< CHANGE
+      - SPRING_DATASOURCE_PASSWORD=1234 # <<< CHANGE (Use secrets in real apps!)
+
+    networks:
+      - monitoring_net
+      - default
+
+networks:
+  monitoring_net:
+    external: true
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)